### ISSUE TITLE
Can't display the number of views for each post

### ISSUE THREAD

**Issue by lei2rock** [theme-next/hexo-theme-next#986]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 

display the number of views for each post
![](https://user-images.githubusercontent.com/19180725/61578328-a3b2f780-ab27-11e9-9edd-0ec86797e4c0.png)

### Actual behavior 

Can't display the number of views for each post, and the browser console shows `The Uncaught ReferenceError: $ is not defined`
![](https://user-images.githubusercontent.com/19180725/61578360-3358a600-ab28-11e9-943e-17f138030fee.jpg)
![](https://user-images.githubusercontent.com/19180725/61578362-35bb0000-ab28-11e9-86b0-a0eeaf1cca9b.jpg)

### Steps to reproduce the behavior 
1. N/A
2. N/A
3. N/A

* Link to demo site with this bug: https://blog.dlzhang.com
* Link(s) to source code or any usefull link(s): https://github.com/lei2rock/blog

### Node.js and NPM Information

```
v12.1.0
6.9.0
```

### Package dependencies Information

https://github.com/lei2rock/blog/blob/src/package.json

#### Hexo version

```
hexo: 3.9.0
hexo-cli: 2.0.0
os: Darwin 18.6.0 darwin x64
node: 12.1.0
v8: 7.4.288.21-node.16
uv: 1.28.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 72
nghttp2: 1.38.0
napi: 4
llhttp: 1.1.1
http_parser: 2.8.0
openssl: 1.1.1b
cldr: 35.1
icu: 64.2
tz: 2019a
unicode: 12.1
```

#### Hexo Configuration

https://github.com/lei2rock/blog/blob/src/_config.yml

### NexT Information

**NexT Version:**

- [x] Latest Master branch
- [ ] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [ ] All schemes
- [ ] Muse
- [ ] Mist
- [ ] Pisces
- [x] Gemini

#### NexT Configuration:

https://github.com/lei2rock/blog/blob/src/source/_data/next.yml

### Other Information

**Comment by stevenjoezhang** [comment]
Fixed in https://github.com/theme-next/hexo-theme-next/commit/10ecc09f36beefaa7218f3f75e5cfe68fcc86234

**Comment by lei2rock** [comment]
> Fixed in [10ecc09](https://github.com/theme-next/hexo-theme-next/commit/10ecc09f36beefaa7218f3f75e5cfe68fcc86234)

Great! It was solved. Thx
